02_getting_started.asciidoc

```diff
@@ -9,6 +9,25 @@ The Lightning Network is accessed via software applications that can speak the L
 
 Users have the highest degree of control by running their own Bitcoin node and Lightning node. However, Lightning nodes can also use a lightweight Bitcoin client (commonly referred to as Simplified Payment Verification (SPV)) to partially validate the correctness of their blockchain.
 
+=== Lightning Explorer
+
+A Lightning network explorer is an useful tool to show the statistics of nodes, channels and network capacity.
+
+The below is an inexhaustive list (in alphanumerical order):
+
+* https://1ml.com/, 1ML Lightning explorer
+* https://explorer.acinq.co/, ACINQ's Lightning explorer, with fancy visualization
+* https://lightblock.me/, Lightblock Lightning explorer
+* https://www.robtex.com/lightning/node/, Robtex Lightning explorer
```

Comment:
this one redirects to this URL https://hashxp.org/lightning/node/

Reply:
@HAOYUatHZ please fix the comment by @grunch and we are ready to merge

Reply:
no problem! I have already fix it

Reply:
@aantonop

Reply:
thanks (: